x/poolmanager/router_test.go

```diff
@@ -1624,6 +1626,19 @@ func (s *KeeperTestSuite) TestEstimateTradeBasedOnPriceImpact() {
 			expectedInputCoin:  sdk.NewCoin(assetBaz, sdk.NewInt(39_947)),
 			expectedOutputCoin: sdk.NewCoin(assetBar, sdk.NewInt(59_327)),
 		},
+		"valid balancer pool - estimate trying to trade 1 token": {
```

Comment:
Thing is this test, would have passed without the recent changes because of the default balancer pool used. In reality what failed was one with strange weights.

Reply:
Resolving this as we discussed it